app/views/runs/show.slim

```diff
@@ -18,15 +18,15 @@
 = render partial: 'runs/full_timeline', locals: {timing: timing, run: @run, compare_runs: @compare_runs}
 
 - if @compare_runs.none? || (@compare_runs.any? && current_user.try(:patron?, tier: 3))
-  - if [@run, *@compare_runs].map(&:video_url).compact.any?
+  - if [@run, *@compare_runs].map(&:video).compact.map(&:url).compact.any?
```

Comment:
For the future when `@compare_runs` can be any number of runs, this turns into an n+1 query for all those videos. Can it be eliminated?